Standards/scs-0118-v1-rook-decision.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Replacement of the deprecated ceph-ansible tool
+type: Decision Record
+status: Draft
+track: IaaS
+---
+
+## Abstract
+
+This decision record evaluates the choice for a modern, future-proof deployment tool for the networked storage solution Ceph in the SCS reference implementation, [OSISM](https://osism.tech/).
+The new deployment tool aims to enhance Kubernetes integration within SCS, potentially allowing users to manage the Ceph cluster with greater ease and efficiency.
```

Comment:
Just a small nit:

I assume with "users" you intend to refer to CSP personnel as in those who *use* OSISM as orchestration tooling such as admins and operators?

When reading across the standards, the term "user" is also often used for SCS users (as in customers) that use the infrastructure. Maybe we should consider a different term here to avoid confusion?

Reply:
I changed the term `users` to `providers` in this document.